### PR TITLE
Added Legendary Games - Treasury of the Pharaohs

### PR DESCRIPTION
**Pull request by axxroytovu**
Added the Treasury of the Pharaoh's AP add-on by Legendary Games

### CODE REVIEW

**Comment by LegacyKing** [comment]
@Nylanfs - I need a PR Review assist stat - Do we have permission from the publisher to include this? And does it meet OGL requirements. Thanks!

File: data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/_treasury_ot_pharaohs.pcc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# CVS $Revision a.1$ $Author Axxroy$ -- Sun Feb 25 14:45:00 2017 -- reformated by prettylst.pl v1.35 (build 553)
+CAMPAIGN:Legendary - Treasury of the Pharaohs
+GAMEMODE:Pathfinder
+RANK:9
```
Comment (LegacyKing) [inline data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/_treasury_ot_pharaohs.pcc]: Improper rank here, you need to go lower. Ranks 7, 8, and 9 are all Homebrew ranks, For a 3rd party, the ranks are 4, 5 and 6.  4 is a Core Rule set, 5 is supplements, and 6 would be accessory supplements.

File: data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/_treasury_ot_pharaohs.pcc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+PRECAMPAIGN:6,INCLUDESBOOKTYPE=Core Rules,INCLUDES=Ultimate Combat,INCLUDES=Advanced Player's Guide,INCLUDES=Ultimate Equipment,INCLUDES=Ultimate Magic,INCLUDES=Melee Tactics Toolbox
+PUBNAMELONG:Legendary Games
+PUBNAMESHORT:Legendary
+PUBNAMEWEB:http://www.makeyourgamelegendary.com/
+SOURCELONG:Treasury of the Pharaohs
+SOURCESHORT:TotP
+SOURCEWEB:http://www.makeyourgamelegendary.com/
+SOURCEDATE:2015-04
+ISOGL:YES
+COPYRIGHT:Treasury of the Pharaohs, Copyright 2015 by Legendary Games
```
Comment (LegacyKing) [inline data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/_treasury_ot_pharaohs.pcc]: Customary to include all of the OGL Copyrights in this section. The final one is the PCGen which you would include yourself as a Data Monkey. (Need to get your credit in here).

File: data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/totp_equip_magic_items.lst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Signet of the Life Pharaoh																TYPE:Magic.Ring	COST:45200		WT:0	MODS:NO				SOURCEPAGE:p.11		SPELLS:Magic Item|TIMES=3|CASTERLEVEL=1|Cure Light Wounds	SPELLS:Magic Item|TIMES=1|CASTERLEVEL=5|Cure Serious Wounds		BONUS:COMBAT|AC|2|TYPE=Deflection		SPROP:Ring of protection +2. The wearer can cast cure light wounds three times per day as a standard action, or a cure serious wounds once per day. When you use this ability to heal a good-aligned creature, that creature gains a +5 sacred bonus on attack rolls, skill checks, ability checks, and saving throws for 1 round. Once per week, the spirit of the Life Pharaoh can forestall the death of the wearer. Anytime the wearer would be killed by hit point damage, the ring bestows a breath of life upon her. If this prevents the wearer’s death, her head is surrounded by a beatific halo for up to 1 hour, shedding light as a light spell and enabling her to gain a +10 sacred bonus on a single Wisdom or Charisma check or Wisdom- or Charisma-based skill check. Once this bonus is used, the halo disappears and the signet of the Life Pharaoh cannot use any of its healing abilities for one week.
```
Comment (LegacyKing) [inline data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/totp_equip_magic_items.lst]: Just a very minor nit here: SPELLS:Magic Item| - I prefer to know what item is granting the spell(s). I would prefer Magic Item actually be the name of the item for clarity sake. However, that is just a minor nit and *is* not required for this review.

**Comment by LegacyKing** [comment]
Three things:
1) Waiting on @Nylanfs to verify we have publisher permission
2) Second, this should actually go onto our NEWSOURCE Repository for complete review (https://github.com/PCGen/pcgen-newsources)
3) Third, it is always best to submit from a branch, not your master.

File: data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/totp_equip_magic_items.lst (hunk 3 above)
Comment (axxroytovu) [inline data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/totp_equip_magic_items.lst]: Thanks, I didn't know you could do that.

**Comment by axxroytovu** [comment]
Made the updates. I'll send it over to the newsources repository (didn't know that was a thing), should I leave this pull request here or delete it?

**Comment by LegacyKing** [comment]
Just delete... but I'll close. It's all good. We test new sources before we incorporate them into the main release program.

**Pull request closed without merging**